src/nrf_802154_swi.c

```diff
@@ -225,8 +227,19 @@ typedef struct
 
         struct
         {
-            bool * p_result;                              ///< CCA config update request result.
-        } cca_cfg_update;                                 ///< CCA config update request details.
+            bool * p_result; ///< CCA config update request result.
+        } cca_cfg_update;    ///< CCA config update request details.
+
+        struct
+        {
+            bool * p_result; ///< RSSI measurement request result.
+        } rssi_measure;      ///< RSSI measurement request details.
+
+        struct
+        {
+            int8_t * p_rssi;                              ///< RSSI measurement result.
+            bool   * p_result;                            ///< RSSI measurement status.
+        } rssi_get;                                       ///< RSSI measurement result getter details.
```

Comment:
```suggestion
        } rssi_get;                                       ///< Getter details of the RSSI measurement result.
```
Long concatenation of nouns, so we must make it easier to read.

Reply:
Would `Details of the RSSI measurement result getter` be ok? I think it is more accurate

Reply:
or `Details of the getter that retrieves the RSSI measurement result.`

Reply:
The second option sounds best, I think.